src/css/relay.css

```diff
@@ -33,11 +36,51 @@
   font-size: 16px;
 }
 
+.fx-relay-menu-iframe ::-moz-focus-inner,
+.fx-relay-menu-body ::-moz-focus-inner,
 .fx-relay-menu ::-moz-focus-inner,
 .fx-relay-icon ::-moz-focus-inner {
   border: 0;
 }
 
+.fx-relay-menu-body {
+  margin: 0;
+  padding: 0;
+  background: var(--relayGrey10);
+  background-color: var(--relayGrey10);
+  box-sizing: border-box;
+}
+
+.fx-relay-menu-iframe iframe { 
+  border-radius: 4px;
+  background: var(--relayGrey10);
+  background-color: var(--relayGrey10);
+  border: 0;
+  box-shadow: 3px 3px 3px 3px #00000017;
+  animation: fxRelayFadeIn 0.2s ease forwards;
+}
+
+.fx-relay-menu-iframe { 
+  width: 300px;
+  min-height: 60px;
+  top: 47px;
+  right: -15px;
```

Comment:
:magic_wand: would like a comment here about where these numbers came from :)

Reply:
Comment added. It's the magic that correctly positions the box underneath the logo/triangle: 
<img width="352" alt="image" src="https://user-images.githubusercontent.com/2692333/160976493-fe0dab76-bc83-43d5-ba68-90a9616300a3.png">